src/webui/www/private/scripts/client.js

```diff
@@ -631,10 +631,8 @@ window.addEvent('load', function() {
                     if (response['trackers']) {
                         for (const tracker in response['trackers']) {
                             const torrents = response['trackers'][tracker];
-                            const host = getTrackerHost(tracker);
-                            const hash = genHash(host);
-                            trackerList.set(hash, {
-                                url: host,
+                            trackerList.set(tracker, {
+                                url: tracker,
```

Comment:
doesn't work as expected... 
I have 2 torrents. torrentA have tracker http://host/url1. torrentB have tracker http://host/url2.
However the webui only shows `host (1)`.